src/lein_tools_deps/env.clj

```diff
@@ -24,12 +24,23 @@
       (or exe (throw (ex-info "Could not find clojure executable" {:tried-paths clojure-executables}))))
     "clojure"))
 
+(defn- windows?
+  []
+  (clojure.string/includes? (System/getProperty "os.name") "Windows"))
+
+(defn- cross-platform-sh
+  [exe & args]
+  (if (windows?)
+    (let [{:keys [out] :as result} (apply shell/sh "cmd" "/c" exe args)]
```

Comment:
I've not used windows for 19+ years; can you tell me what `cmd /c` is doing?

Reply:
It runs a command inside the windows command line. It's like `sh -c` on unix. It seems to be needed for this to work on windows